lib/jekyll/assets/liquid/filters.rb

```diff
@@ -9,7 +9,7 @@ module Assets
     module Liquid
       module Filters
         ACCEPTABLE_FILTERS = [:css, :img, :asset_path, :stylesheet,
-          :javascript, :style, :img, :js]
+          :javascript, :style, :image, :js]
```

Comment:
I'm fine if you wish to add "image", I am not fine with the removal of "img", please keep both.  Thanks!